inspirehep/base/static/js/citation_modal.js

```diff
@@ -40,6 +40,14 @@ define(
           $("#text" + data.recid).text(data.result);
           $("#format" + data.recid).text('BibTex')
           $("#download" + data.recid).attr("href", "/formatter/download-bibtex/" + data.recid)
+          setTimeout(function(){
```

Comment:
Tried it, wasn't working for me :/ Also tried $('.editable').trigger('click'). No luck!